app/assets/javascripts/users/gardens.js

```diff
@@ -22,6 +22,8 @@ openFarmApp.controller('gardenCtrl', ['$scope', '$http', '$rootScope',
           function(success, response) {
             if(success) {
               $scope.profileUser.gardens = response;
+              console.log(response);
```

Comment:
Hey @simonv3 ! I have added a new commit with the console.log statements removed and a description added. Coveralls seems okay with it. :dancer: